### PR TITLE
fix radio button on ie11

### PR DESCRIPTION
**Pull request by dprzybylek**

### CODE REVIEW

File: src/components/RadioButton/style.scss [hunk 1]
```diff
@@ -51,7 +52,7 @@ $base-class: 'radio';
     font-size: 15px;
     color: $font-primary-color;
     line-height: 20px;
-    flex: 1;
+    flex: 1 0 auto;
```
Comment (kamilmateusiak) [inline src/components/RadioButton/style.scss]: can you fix `.lc-checkbox__text` as well? there is the same problem

Comment (kamilmateusiak) [reply src/components/RadioButton/style.scss]: and `.lc-checkbox__label`

**Comment by kamilmateusiak** [comment]
After the merge, it would be great to test it with beta release. In AA we are using RadioButtons and AFAIR there are some hacks which we need to fix

File: src/components/RadioButton/style.scss (hunk 1 above)
Comment (dprzybylek) [reply src/components/RadioButton/style.scss]: @kamilmateusiak done

**Pull request merged**